src/Bridge/Twig/Extension/EnumExtension.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Greg0ire\Enum\Bridge\Twig\Extension;
+
+use Greg0ire\Enum\AbstractEnum;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+final class EnumExtension extends \Twig_Extension
+{
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    /**
+     * @param TranslatorInterface $translator
+     */
+    public function __construct(TranslatorInterface $translator = null)
+    {
+        $this->translator = $translator;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFilters()
+    {
+        return [
+            new \Twig_SimpleFilter('enum_label', [$this, 'label']),
+        ];
+    }
+
+    /**
+     * Displays the label corresponding to a specific value of an enumeration.
+     *
+     * @param mixed       $value             Must exist in the enumeration class specified with $class
+     * @param string      $class             The enum class name
+     * @param string|bool $translationDomain The translation domain to use if the translator if available.
+     *                                       string: Use the specified one
+     *                                       null: Use the default one
```

Comment:
> Anyway, AFAIK, this is not the role of this library to decide in which catalog the end user should put the translation. :wink: 

You're right again, and when doing my use case, I'll introduce a `TranslationDomainResolverInterface` or something like that, so that one can implement its own.